packages/data/test/repositories/auth_repository_impl_test.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:data/repositories/auth_repository_impl.dart';
+import 'package:domain/entities/user_entity.dart';
+import 'package:domain/exceptions/auth_failure_exception.dart';
+import 'package:domain/services/auth_service.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mocktail/mocktail.dart';
+
+class MockAuthService extends Mock implements AuthService {}
+
+void main() {
+  const mockLogin = 'login';
+  const mockPassword = 'password';
+
+  late MockAuthService mockAuthService = MockAuthService();
+  late AuthRepositoryImpl authRepository = AuthRepositoryImpl(
```

Comment:
1) why late?
2) why `AuthRepositoryImpl` and not `AuthRepository`?

Reply:
1. I forgot to remove late after some experiments with refactoring.
2. Fixed.